buildtest/menu/build.py

```diff
@@ -166,6 +162,15 @@ def func_build_subcmd(args):
         ]
         logger.debug(f"Configuration List after applying exclusion: {config_files}")
 
+        # if no files remain after exclusion let's stop now.
+        if not config_files:
```

Comment:
Just to check here - you don't want any message printed to the log, but you are passing to sys.exit?

Reply:
yes that's right, buildtest is there to inform user after discovery or after exclusion step if there are no configs. I know it's repeated twice after each stage. Atleast user should know what buildtest is trying to do.